src/pdf.sandbox.js

```diff
@@ -102,8 +104,19 @@ class Sandbox {
 }
 
 function QuickJSSandbox() {
+  const handlers = {
+    callFunction(name, args) {},
+    clone(obj) {
+      return window.JSON.stringify(obj);
+    },
+  };
+  window.sandboxExtra = buildExtra(window, handlers);
```

Comment:
Without a comment, it wasn't obvious at all why the implementation is like this.

But here it became apparent....: https://github.com/mozilla/pdf.js.quickjs/blob/f927cb7527c7cf0087872720aa151e7d4cf43460/src/myjs.js#L37-L46

Please add an explanatory comment when something non-obvious is done.